Formula/wpscan.rb

```diff
@@ -0,0 +1,55 @@
+class Wpscan < Formula
+  desc "Black box WordPress vulnerability scanner"
+  homepage "https://wpscan.org"
+  url "https://github.com/wpscanteam/wpscan/archive/2.9.2.tar.gz"
+  sha256 "01c888bf2310354823a9522cd929e0caad5570c14eef6f47754c90e268fccad8"
+  head "https://github.com/wpscanteam/wpscan.git"
+
+  depends_on :ruby => "2.1.9"
+
+  def install
+    inreplace "lib/common/common_helper.rb" do |s|
+      s.gsub! "ROOT_DIR, 'cache'", "'#{var}/cache/wpscan'"
+      s.gsub! "ROOT_DIR, 'log.txt'", "'#{var}/log/wpscan/log.txt'"
+    end
+    (var/"log/wpscan").mkpath
+
+    if build.stable?
+      # Remove when a version >2.9.2 is released.
+      inreplace "lib/environment.rb", "require 'xmlrpc/client'", ""
+    end
+
+    system "unzip", "-o", "data.zip"
+    libexec.install "data", "lib", "spec", "Gemfile", "wpscan.rb"
+    if build.head?
+      # Merge into stable when a version >2.9.2 is released.
+      libexec.install "Gemfile.lock"
+    end
+
+    ENV["GEM_HOME"] = libexec
+    ENV["BUNDLE_PATH"] = libexec
+    ENV["BUNDLE_GEMFILE"] = libexec/"Gemfile"
+    system "gem", "install", "bundler"
+    system libexec/"bin/bundle", "install", "--without", "test"
+
+    (bin/"wpscan").write <<-EOS.undent
+      #!/bin/bash
+      GEM_HOME=#{libexec} BUNDLE_GEMFILE=#{libexec}/Gemfile \
+        exec #{libexec}/bin/bundle exec ruby #{libexec}/wpscan.rb "$@"
+    EOS
+  end
+
+  def post_install
+    system bin/"wpscan", "--update"
```

Comment:
`wpscan --update` updates the databases of vulnerabilities and stuff (forgot a comment here). The exit status is zero even if you're already up-to-date. I suppose it will exit non-zero if you're not connected to the Internet, but since this is `post_install` does it really matter?